### ISSUE TITLE
Revise Variant form is broken

### ISSUE THREAD

**Issue by obigriffith** [griffithlab/civic-v2#423]
Right now there are several problems with the Revise Variant form. 

If I revise variant from scratch (new variant with no data yet) there are problems that prevent completing the revision. (1) Variant Description is required but should be optional. (2) The Ref and Var bases fields are missing. So, there is no way to actually specify even a simple variant. (3) When you add an hgvs string the whole string is not visible without scrolling inside small box which is not ideal for checking your work before hitting submit. (4) Ensembl version does not allow "75" which is the most common (standard) choice. (5) I think that for a "new" variant. The first time adding anything after a variant is auto-created. A comment should maybe be optional. 

If I revise an existing variant (e.g., V600E) that is fully completed. All the fields appear empty rather than being pre-populated with the current data.

![screencapture-civicdb-org-variants-2989-revise-suggest-2022-04-10-17_51_50](https://user-images.githubusercontent.com/1638042/162643353-c5083dc0-d600-42a6-85ea-a196da39b1af.png)

![screencapture-civicdb-org-variants-12-revise-suggest-2022-04-10-17_52_15](https://user-images.githubusercontent.com/1638042/162643359-09b7b542-ab06-47ab-ba8f-28d8c2968c34.png)

**Comment by obigriffith** [comment]
Also, Submit button is greyed out and doesn't allow submission even with currently available/required fields completed.

**Comment by susannasiebert** [comment]
This is related to: https://github.com/griffithlab/civic-v2/issues/245

**Comment by susannasiebert** [comment]
> Also, Submit button is greyed out and doesn't allow submission even with currently available/required fields completed.

Did you add a comment (10 characters min length)?

**Comment by obigriffith** [comment]
Yes. In the first screenshot I included above you can see I added a comment of ~20chars but the submit button remains greyed out.

**Comment by susannasiebert** [comment]
This is for us internally to track which issues have been fixed (locally, not on production):
- [x] Variant Description is required but should be optional. 
- [ ] The Ref and Var bases fields are missing. 
- [ ] When you add an hgvs string the whole string is not visible without scrolling inside small box which is not ideal for checking your work before hitting submit. 
- [ ] Ensembl version does not allow "75" which is the most common (standard) choice. 
- [ ] I think that for a "new" variant. The first time adding anything after a variant is auto-created. A comment should maybe be optional.
- [ ] Form doesn't populate existing data
- [ ] Submit button greyed out even if all required fields are provided

**Comment by obigriffith** [comment]
Just to confirm. I reviewed all these issues on live/deployed and they indeed all appear to be fixed
